crates/astria-composer/src/config/searcher.rs

```diff
@@ -0,0 +1,119 @@
+use std::{
+    net::{
+        AddrParseError,
+        SocketAddr,
+    },
+    str::FromStr,
+};
+
+use figment::{
+    providers::{
+        Env,
+        Serialized,
+    },
+    Figment,
+};
+use serde::{
+    Deserialize,
+    Serialize,
+};
+use thiserror::Error;
+
+use super::cli;
+use crate::types::rollup::ChainId;
+
+const DEFAULT_SEQUENCER_URL: &str = "127.0.0.1:1317";
+const DEFAULT_API_URL: &str = "127.0.0.1:8080";
```

Comment:
which API?

Reply:
right now it's just `/healthz`, this should be changed to just a port number tho.

Reply:
ok, so it's the API of the composer itself. cool